services/dedup/dedup.go

```diff
@@ -111,25 +111,17 @@ func (d *DedupHandleT) openBadger() {
 
 	opts := badger.
 		DefaultOptions(d.path).
-		WithTruncate(true).
```

Comment:
I can see that `v3` doesn't have `WithTruncate()` func. Can you please clarify if it changes anything for us, since we are not setting it explicitly?